src/frontends/tensorflow/src/variables_index.hpp

```diff
@@ -89,8 +103,20 @@ class VariablesIndex {
     /// \param shard_id Requested shard_id
     /// \returns Valid shared_ptr with ifstream or with nullptr if shard isn't found
     std::shared_ptr<std::ifstream> get_data_file(const int32_t shard_id) const {
```

Comment:
I think it makes sense to add TF FE unit-test(s) to check both modes (w/wo mmap) on some model that is of SM format with some constants and variables.
You just need to load them differently with boolean flag

Reply:
Not sure I got your point where I can pass this flag...

Reply:
`load` method of FE class can accept boolean flag in arguments that indicates to switch on/off mmap. So I suggest to extend the current TF FE unit-tests to vary this.